tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_ConfiguredAssetFileSystemExample.ipynb

```diff
@@ -0,0 +1,1129 @@
+       "      <td>2.0</td>\n",
+       "      <td>2020-12-27 10:43:42</td>\n",
+       "      <td>2020-12-27 10:51:05</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.22</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>N</td>\n",
+       "      <td>163</td>\n",
+       "      <td>48</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>7.0</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.5</td>\n",
+       "      <td>2.06</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.3</td>\n",
+       "      <td>12.36</td>\n",
+       "      <td>2.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>2.0</td>\n",
+       "      <td>2020-12-08 13:42:52</td>\n",
+       "      <td>2020-12-08 13:54:45</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.84</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>N</td>\n",
+       "      <td>137</td>\n",
+       "      <td>229</td>\n",
+       "      <td>2.0</td>\n",
+       "      <td>9.0</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.5</td>\n",
+       "      <td>0.00</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.3</td>\n",
+       "      <td>12.30</td>\n",
+       "      <td>2.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>2.0</td>\n",
+       "      <td>2020-12-19 11:56:43</td>\n",
+       "      <td>2020-12-19 12:08:43</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>1.55</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>N</td>\n",
+       "      <td>24</td>\n",
+       "      <td>74</td>\n",
+       "      <td>1.0</td>\n",
+       "      <td>9.5</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.5</td>\n",
+       "      <td>2.58</td>\n",
+       "      <td>0.0</td>\n",
+       "      <td>0.3</td>\n",
+       "      <td>12.88</td>\n",
+       "      <td>0.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   vendor_id      pickup_datetime     dropoff_datetime  passenger_count  \\\n",
+       "0        2.0  2020-12-15 12:20:27  2020-12-15 12:40:49              4.0   \n",
+       "1        2.0  2020-12-28 12:51:25  2020-12-28 13:15:12              1.0   \n",
+       "2        2.0  2020-12-27 10:43:42  2020-12-27 10:51:05              1.0   \n",
+       "3        2.0  2020-12-08 13:42:52  2020-12-08 13:54:45              1.0   \n",
+       "4        2.0  2020-12-19 11:56:43  2020-12-19 12:08:43              1.0   \n",
+       "\n",
+       "   trip_distance  rate_code_id store_and_fwd_flag  pickup_location_id  \\\n",
+       "0           5.76           1.0                  N                 209   \n",
+       "1          11.64           1.0                  N                 161   \n",
+       "2           1.22           1.0                  N                 163   \n",
```

Comment:
@Shinnnyshinshin Would it be acceptable to clear the output to aide with the review, or are we supposed to commit to GitHub after the execution to show that it was done successfully?  Thanks.

Reply:
Great point. I do think the output should be cleared :)